src/Client/AbstractClient.php

```diff
@@ -63,6 +63,15 @@ protected function getRequestHeaders(): array
         ];
     }
 
+    protected function getRequestHeadersBasicAuth($username, $password): array
```

Comment:
This one was also reverted, see the mentioned pull request here: https://github.com/btcpayserver/btcpayserver-greenfield-php/pull/101/files#diff-0802f9ba4fe10ad4468ad6c06f332eda04f8455467646cf2adcbb2b05a14c461L66